tools/deno_http_proxy.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
Can we call it deno_proxy and node_proxy ? "deno_http_proxy" is quite long for the UI